card/card.css

```diff
@@ -0,0 +1,147 @@
+.card{
+    display: flex;
+    position: relative;
+    font-family: 'Source Serif 4', sans-serif;
+    color: var(--color-complementary);
+    background-color: #272626;
+    background: rgba(255, 255, 255, 0.2);
+    margin: var(--spacing-medium);
+    border-radius: var(--bradius-small);
+    box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
+}
+
+.card img{
+    max-height: 20rem;
+    min-height: 100%;
+} 
+
+.card__content *{
+    margin: 2px;
+}
+
+.card-horizontal{
+    max-width: 70rem;
+    min-width: min-content;
+    max-height: fit-content;
```

Comment:
What I think, there is no need for min-width and max-height 
1. It has no effect. The content will itself will take the minimum required width 
2. you are using media query so it no it turns into a vertical card and hence this code never gets applied.
3. Even max-height has no effect as the content will itself take minimum space required.